content/en/blog/2018-10-19-helm-turns-five.md

```diff
@@ -0,0 +1,40 @@
+---
+title: " Helm Turns 5, and GitHub Gives the Gift of Charts"
+slug: "helm-turns-five"
+authorname: "Matt Butcher & Matt Farina"
+author: "@technosophos & @mattfarina"
+authorlink: "https://helm.sh"
+date: "2020-10-16"
```

Comment:
This is set to today for preview purposes. 
Will adjust to the 19th once lgtm ✅